backend/tests/E2E/Book/GetBookByIdTest.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+namespace App\Tests\E2E\Book;
+
+use Symfony\Bundle\FrameworkBundle\KernelBrowser;
+
+class GetBookByIdTest extends \Symfony\Bundle\FrameworkBundle\Test\WebTestCase
+{
+    private KernelBrowser $client;
+    private int $id;
+
+    protected function setUp(): void
+    {
+        $this->client = static::createClient();
+        $headers = ['CONTENT_TYPE' => 'application/json'];
+        $body = '{
+                "title": "Titolo di test",
+                "author": "Autore di test",
+                "price": 20.99,
+                "description": "Test inserimento"
+            }';
+        $this->client->request('POST', '/api/books', [], [], $headers, $body);
+        $this->id = json_decode($this->client->getResponse()->getContent())->id;
```

Comment:
suggestion: in questo caso potrebbe far comodo tirarsi dentro l'entityManager di Symfony e persistere l'entità tipo:

 ```
protected EntityManagerInterface|null $manager;
 
 protected function setUp(): void
    {
        $this->client = static::createClient();
        $this->manager = static::getContainer()->get(EntityManagerInterface::class);
        $book = new Boook(.....)
        
        $this->manager->persist($book);
        $this->manager->flush();
    }
```
    
nel caso lo vediamo domani